dags/utils/gcp.py

```diff
@@ -500,6 +503,10 @@ def bigquery_etl_query(
         + ["--dataset_id=" + dataset_id]
         + (["--project_id=" + project_id] if project_id else [])
         + ["--parameter=" + parameter for parameter in parameters]
+        + [
+            "--schema_update_option={{ 'ALLOW_FIELD_ADDITION' if ds in %r else '' }}"
+            % (allow_field_addition_dates,)
+        ]
```

Comment:
Can we wrap this in `([ ...] if allow_field_addition_dates else [])`?

Reply:
yeah